include/deal.II/numerics/vector_tools.h

```diff
@@ -919,13 +919,18 @@ namespace VectorTools
    * @endcode
    * where <code>qp_data</code> is a an object of type Table<2, VectorizedArray<double> >,
    * which stores quadrature point data.
+   *
+   * @p fe_component allow to additionally specify which component of @p data
+   * to use in case it was constructed with an <code>std::vector<const DoFHandler<dim>*></code>
+   * in will be internally used in constructor of FEEvaluation object.
```

Comment:
I come late to this party, but would you mind fixing the grammar here? "...to use in case it was constructed with an X ***in will be internally...***" does not make sense to me. Do you mean "...to use in case it was constructed with an X. ***It will be internally...***"

For reasons I can't articulate, the English language also likes "will be used internally" better than "will be internally used".